### ISSUE TITLE
How to use Network.enable

### ISSUE THREAD

**Issue by popouille** [rialto-php/puphpeteer#133]
Hello, i trying to implement the following code in the puphpeteer way :
```
await page._client.send('Network.enable', {
      maxResourceBufferSize: 1024 * 1204 * 100,
      maxTotalBufferSize: 1024 * 1204 * 200,
    })
```

i tested:
`$page->_client->send('Network.enable', ['maxResourceBufferSize '=> 1024 * 1204 * 100, 'maxTotalBufferSize' => 1024 * 1204 * 200]);`

But i got this error:

`Page._client is not a function`

I can't figure out with the current documentation how to do it.

**Comment by nesk** [comment]
This is an undocumented property so I can't tell _when_ it was added to Puppeteer. However, my guess is that the property is not available in the version used by PuPHPeteer (Puppeteer 5.5 if you use the latest version of PuPHPeteer).

**Comment by nesk** [comment]
Well, I've just tried, it works:

```
❯ psysh
Psy Shell v0.10.5 (PHP 7.4.13 — cli) by Justin Hileman
New version is available (current: v0.10.5, latest: v0.10.6)
>>> $pu = new Nesk\Puphpeteer\Puppeteer()
=> Nesk\Puphpeteer\Puppeteer {#2580}
>>> $br = $pu->launch()
=> Nesk\Puphpeteer\Resources\Browser {#2578}
>>> $pa = $br->newPage()
=> Nesk\Puphpeteer\Resources\Page {#2594}
>>> $pa->_client
=> Nesk\Puphpeteer\Resources\CDPSession {#2596}
>>>
```

Just update to the latest version of PuPHPeteer, run `npm update` to also update the Puppeteer version used, and it should work :)
